framework/src/base/MooseApp.C

```diff
@@ -1509,14 +1508,10 @@ MooseApp::createMinimalApp()
     // Build the Action parameters
     InputParameters action_params = _action_factory.getValidParams("CreateProblemAction");
     action_params.set<std::string>("type") = "FEProblem";
+    action_params.set<bool>("skip_param_construction") = true;
```

Comment:
You are adding "skip_param_construction" parameter to `InputParameters` of `CreateProblemAction`, but the parameter was not added via `addParamXYZ`, so @rwcarlsen's thing should bomb out with an error message like setting unregistered parameter. Why is that not happening?

Reply:
That check only occurs for `MooseObject` subclasses - since `MooseApp` isn't, its params aren't subject to that check.

Reply:
To be more clear - the check occurs in the moose object factory.  I suppose we could put a similar check in the app factory.

Reply:
You mean action factory?

Reply:
yes - I meant action factory.